biggraphite/glob_utils.py

```diff
@@ -281,13 +281,15 @@ class GlobDirectoryResult:
     def __init__(self, name):
         """Create new instance of GlobMetricResult."""
         self.name = name
-        self.value = None
```

Comment:
`GlobDirectoryResult` and `GlobMetricResult` are meant to be used indifferently and thus should have the same API. Removing it could lead to unexpected `AttributeError`. 

Maybe we could make it more clear / obvious... as there is no typing opportunity for now.

Reply:
ok that explains why :)

I will add a comment to make it clear